app/assets/stylesheets/application.scss

```diff
@@ -1,44 +1,17 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+@import 'font-awesome';
 
-$input-placeholder-color: hsl(0, 0%, 86%); //has-text-grey-lighter
```

Comment:
i think i still need this on the rails side. it's a bulma variable so my placeholder text isn't black in rails forms